code/src/java/pcgen/cdom/facet/analysis/VariableFacet.java

```diff
@@ -90,13 +90,8 @@ public void dataRemoved(DataFacetChangeEvent<CharID, CDOMObject> dfce)
 	private void add(CharID id, VariableKey vk, Formula formula, CDOMObject cdo)
 	{
 		Map<VariableKey, Map<Formula, Set<CDOMObject>>> map = getConstructingCachedMap(id);
-		Map<Formula, Set<CDOMObject>> subMap = map.get(vk);
-		if (subMap == null)
-		{
-			subMap = new HashMap<>();
-			map.put(vk, subMap);
-		}
-		Set<CDOMObject> sources = subMap.get(formula);
+        Map<Formula, Set<CDOMObject>> subMap = map.computeIfAbsent(vk, k -> new HashMap<>());
```

Comment:
Whitespace not consistent

Reply:
Would be nice to expand `vk` to `variableKey` as well.  And `map` to a more meaningful name (what is it a map of?), same with `subMap` etc.  Part of the challenge of the CDOM framework code is that it's very abstract, so variable names that describe the semantics in PCGen terms helps.